### PR TITLE
tests: add details for some snap connection command line tests

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

File: tests/main/snap-connect/task.yaml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 summary: Check that snap connect works
 
+details: |
+    This test verifies multiple way to invoke "snap connect". It
```
Comment (sergiocazzolato) [inline tests/main/snap-connect/task.yaml:4]:
```suggestion
    This test verifies multiple ways to invoke "snap connect". It
```

File: tests/main/snap-connect/task.yaml [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+    checks proper errors are reported. It makes sure slot in some
+    contexts can be implicit. Calls are indempotent. It tests
```
Comment (sergiocazzolato) [inline tests/main/snap-connect/task.yaml:6]: indempotent?

**Review by sergiocazzolato — APPROVED** [review]
Minor commens inline. Thanks

File: tests/main/snap-connect/task.yaml (hunk 2 above)
Comment (valentindavid) [reply tests/main/snap-connect/task.yaml:6]: Connecting an already connected connection does not do anything.

Comment (valentindavid) [reply tests/main/snap-connect/task.yaml:6]: idempotent. Sorry, that was a typo.

**Pull request merged**